cluster-scope/overlays/prod/moc/smaug/kustomization.yaml

```diff
@@ -32,6 +32,7 @@ resources:
 - ../../../../base/core/namespaces/ai-services-thanos-grpc
 - ../../../../base/core/namespaces/apicurio-apicurio-registry
 - ../../../../base/core/namespaces/b4mad-racing
+- ../../../../base/core/namespaces/connaisseur/namespace.yaml
```

Comment:
```suggestion
- ../../../../base/core/namespaces/connaisseur
```

Reply:
@harshad16 sorry about that...too much context switching today